analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -167,6 +172,25 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
                         [], [], [error], [])
             }
 
+            project.gradle.getIncludedBuilds().each {
+                project.logger.warn("Processing included build '$it.name, $it.projectDir'.")
```

Comment:
Maybe use `quiet` instead of `warn` as from the message it's a bit unclear why this should be a warning. Or add some of the explanation from the comment below, e.g.:
```kotlin
"Processing included build '$it.name' from directory '$it.projectDir.path'. This requires use of internal Gradle API and might fail if the project uses an incompatible Gradle version."
```

Reply:
Well I used `.warn` everywhere in this commit because this is invaluable information to debug composite build processing. Currently nothing lower than `WARN` is logged in gradle.log. Since ORT does not allow to change Gradle logging level, the only way to access this information would be to change ORT source code and rebuild the Docker image to configure Gradle logging.

Maybe a later commit will align Gradle logging with ORT log level or add a parameter in ORT to control it. Until then, I would prefer we always log this information hence the `.warn`.

Reply:
`error`, `quiet` and `warn` are all enabled by default, see: https://docs.gradle.org/current/userguide/logging.html

Reply:
Understood, I was abused by the name `quiet`... Thanks for the link !